src/ts/components/uicontainer.ts

```diff
@@ -53,7 +53,12 @@ export class UIContainer extends Container<UIContainerConfig> {
 
     this.config = this.mergeConfig(config, <UIContainerConfig>{
       cssClass: 'ui-uicontainer',
-      hideDelay: 5000,
+      hideDelay: 2000,
+      hidePlayerStateExceptions: [
```

Comment:
This will impact customers' custom UIs so I'd rather go the safe way and do this in our default UI structure only (and schedule the general change for v4).

Reply:
> ... in our default UI structure ...

Do you mean only within `UIFactory`?

I thought about the same but decided against as we want to change the default behaviour in all UI variants. But thinking again it would make more sense to not change components behaviour.

Reply:
Yes that's what I meant :)

Reply:
But thinking about it a third time maybe it's better to have it here 🤔 
My argument for this is that the value in `UIFactory` would override this one here. So if a customer changed it already within the `UIContainer` he would need to remove it again from the `UIFactory` (all variants). 

On the other side if a customer doesn't want this change he just needs to change it here once 🤔 

Maybe both variants have advantages and disadvantages. Not sure what's the better place to do this 'breaking' change.